src/components/GifDetailView/GifDetailView.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Component to show target gif's original file
+ */
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import favoritesActions from '../../actions/favorites';
+
+import { FavButton } from '../ListGifs/ListGifs.styled';
+import { GifDetailViewStyled } from './GifDetailView.styled';
+
+class GifDetailView extends Component {
```

Comment:
this component should be a stateless one, cause it doesn't have lifecycle

Reply:
yup, my bad, sorry